<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LinearDeauthTab</class>
 <widget class="QWidget" name="LinearDeauthTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>420</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>420</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>420</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="apListLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>AP List</string>
   </property>
  </widget>
  <widget class="QLabel" name="interfaceListLabel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>40</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Interface List</string>
   </property>
  </widget>
  <widget class="QListWidget" name="apListWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>161</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="cout">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>70</y>
     <width>271</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>240</y>
     <width>271</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="rescanApsButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Rescan APs</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rescanInterfacesButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>270</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Rescan Interfaces</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveCaptureButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>310</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Capture</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>270</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>10</y>
     <width>84</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="captureButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>310</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Capture</string>
   </property>
  </widget>
  <widget class="QListWidget" name="interfaceListWidget">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>161</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="commentTextEdit">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>360</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="commentLabel">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>340</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Comment</string>
   </property>
  </widget>
  <widget class="QListWidget" name="connectedUserList">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>331</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="rescanTargetsButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>270</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Rescan Targets</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Connected Targets List</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
